tsfile/src/main/java/org/apache/iotdb/tsfile/common/conf/TSFileConfig.java

```diff
@@ -292,6 +292,26 @@ public void setEndian(String endian) {
    */
   private String hdfsPort = "9000";
 
+  /**
+   * Default DFS NameServices is hdfsnamespace
+   */
+  private String dfsNameServices = "hdfsnamespace";
+
+  /**
+   * Default DFS HA name nodes are nn1 and nn2
+   */
+  private String dfsHaNamenodes = "nn1,nn2";
+
+  /**
+   * Default DFS HA automatic failover is enabled
+   */
+  private boolean dfsHaAutomaticFailoverEnabled = true;
+
+  /**
+   * Default DFS client failover proxy provider is "org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider"
+   */
+  private String dfsClientFailoverProxyProvider = "org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider";

```

Comment:
I think HDFS related configurations are better in another file.
How do you think? @qiaojialin

Reply:
have no idea